### PR TITLE
Trend Micro Gov Changes

### PR DESCRIPTION
**Pull request by v-ampami**
Fixes #
Trend Micro Gov Changes
## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: DataConnectors/Trend Micro/TrendMicroXDR_REST_API_Connector.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
...
...ploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinel-trendmicroxdr-azuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter a unique **Function Name**, **Workspace ID**, **Workspace Key**, **API Token**, **Region Code** and **logAnalyticsUri (optional)**. \n - Note: Provide the appropriate region code based on where your Trend Micro XDR instance is deployed: us, eu, au, in, sg, jp  \n - Note: If using Azure Key Vault secrets for any of the values above, use the`@Microsoft.KeyVault(SecretUri={Security Identifier})`schema in place of the string values. Refer to [Key Vault references documentation](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) for further details. \n- Use logAnalyticsUri to override the log analytics API endpoint for dedicated cloud. For example, for public cloud, leave the value empty; for Azure GovUS cloud environment, specify the value in the following format: https://<CustomerId>.ods.opinsights.azure.us. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```
Comment (v-jayakal) [inline DataConnectors/Trend Micro/TrendMicroXDR_REST_API_Connector.json:131]: Remove LogAnalytics related changes, its Automatic deployment, not needed.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Trend Micro/TrendMicroXDR_REST_API_Connector.json (hunk 1 above)
Comment (v-rucdu) [reply DataConnectors/Trend Micro/TrendMicroXDR_REST_API_Connector.json]: Done

File: DataConnectors/Trend Micro/azuredeploy_TrendMicroVisionOne_API_FunctionApp.json [hunk 2]
```diff
@@ -174,31 +178,19 @@
...
+                          "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(parameters('FunctionName')),';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(parameters('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
                           "WEBSITE_CONTENTSHARE": "[toLower(parameters('FunctionName'))]",
                           "workspaceId": "[parameters('WorkspaceID')]",
                           "workspaceKey": "[parameters('WorkspaceKey')]",
                           "api_key": "[parameters('APIKey')]",
                           "regioncode": "[parameters('RegionCode')]",
+                          "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
                           "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Trend%20Micro/AzureFunctionTrendMicroXDR.zip?raw=true"
```
Comment (v-jayakal) [inline DataConnectors/Trend Micro/azuredeploy_TrendMicroVisionOne_API_FunctionApp.json:189]: Can we create short url and update this here?

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Trend Micro/azuredeploy_TrendMicroVisionOne_API_FunctionApp.json (hunk 2 above)
Comment (v-rucdu) [reply DataConnectors/Trend Micro/azuredeploy_TrendMicroVisionOne_API_FunctionApp.json]: Done

**Review by nazang — APPROVED** [review]

**Pull request merged**